cmd/entrypoint/main.go

```diff
@@ -64,6 +68,20 @@ func cp(src, dst string) error {
 	return err
 }
 
+func checkForBreakpointOnFailure(e entrypoint.Entrypointer, breakpointExitPostFile string) {
+	if e.BreakpointOnFailure {
+		if waitErr := e.Waiter.Wait(breakpointExitPostFile, false, false); waitErr != nil {
+			log.Println("error occurred while waiting for " + breakpointExitPostFile)
+		}
+		// get exitcode from .breakpointexit
+		exitCode, readErr := e.ReadBreakpointExitCodeFromDisk(breakpointExitPostFile)
```

Comment:
readability nit: `FromDisk` doesn't add much to this method name. Maybe just `BreakpointExitCode`?

Reply:
Updated from ReadBreakpointExitCodeFromDisk to BreakpointExitCode